bedrock/mozorg/views.py

```diff
@@ -304,6 +305,88 @@ def partnerships(request):
     return process_partnership_form(request, 'mozorg/partnerships.html', 'mozorg.partnerships')
 
 
+def process_content_services_form(request, template, success_url_name, template_vars=None, form_kwargs=None):
+    template_vars = template_vars or {}
+    form_kwargs = form_kwargs or {}
+
+    if request.method == 'POST':
+        form = ContentServicesForm(data=request.POST, **form_kwargs)
+
+        msg = 'Form invalid'
+        stat = 400
+        success = False
+
+        if form.is_valid():
+            data = form.cleaned_data.copy()
+
+            honeypot = data.pop('office_fax')
+
+            if honeypot:
```

Comment:
Wouldn't returning an error for the spam trap just tell the spammers that there are spam traps they need to fix? I think it'd be better if we just made it seem like it succeeded and not submit anything to SF.